react/features/dial-out/actions.js

```diff
@@ -47,6 +47,19 @@ export function checkDialNumber(dialNumber) {
     return (dispatch, getState) => {
         const { dialOutAuthUrl } = getState()['features/base/config'];
 
+        if (!dialOutAuthUrl) {
+            // no check url, let's say it is valid
```

Comment:
I would prefer not calling it "check url" because that specific wording is not used elsewhere in this file. Maybe just call it auth url, for example.